gulpfile.js

```diff
@@ -0,0 +1,120 @@
+/* Shoptet Gulpstack, by Klubus Creative */
+
+// Variables
+import { config } from './config.js';
+import path from 'path';
+import gulp from "gulp";
+import pkg from 'gulp-inject-string';
+import less from "gulp-less";
+import autoPrefixer from "gulp-autoprefixer";
+import concat from "gulp-concat";
+import cleanCss from "gulp-clean-css";
+import uglify from "gulp-uglify";
+import javascriptObfuscator from "gulp-javascript-obfuscator";
+import rename from "gulp-rename";
+
+const { wrap } = pkg;
+
+const rootDir = process.cwd();
+
+const sourceFolder = path.join(rootDir, config.sourceFolder ?? 'src');
+const outputFolder = path.join(rootDir, config.outputFolder);
+
+const addonPrefix = config.addonPrefix;
+const addonInfo = config.addonInfo;
+
+const env = config.env;
+const isProduction = (env == "production") ? true : false;
```

Comment:
Tady se nám trochu rozchází nastavení webpacku a gulpu. Webpack očekává `env.production = true|false`. Vy tu pracujete s `env = "production"`. Nejspíš to dělá stejnou věc, ale přijde mi to trochu matoucí.
Ještě jsem se chtěl zeptat, jestli počítáte s `env`, který může uživatel zadat v CLI?